packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -27,6 +27,7 @@ class BondError extends Error {}
 
 class BondWithdrawalWatcher extends BaseWatcher {
   siblingWatchers: { [chainId: string]: BondWithdrawalWatcher }
+  lastAvailableCredit: { [sourceChain: string]: { [destinationChain: string]: BigNumber }} = {}
```

Comment:
If the `lastAvailableCredit` `sourceChain` key is always going to be `this.bridge.chainSlug`, then it can be omitted to make the mapping simple

```js
lastAvailableCredit: { [destinationChain: string]: BigNumber } = {}
```